packages/main/src/plugin/index.ts

```diff
@@ -1069,7 +1069,10 @@ export class PluginSystem {
             containerFile: relativeContainerfilePath,
             tag: imageName,
             platform,
-            provider: selectedProvider,
+            provider: {
+              ...selectedProvider,
+              status: () => selectedProvider.status,
+            },
```

Comment:
here as well we should remove this change and make the method accepting the info object